storage/shard.go

```diff
@@ -51,9 +51,14 @@ import (
 )
 
 const (
-	shardIterateBatchPercent     = 0.01
-	expireBatchLength            = 1024
-	defaultTickSleepIfAheadEvery = 128
+	shardMinimumTickPerSeriesSleeps        = 32
```

Comment:
a comment on this variable would be good

Reply:
I've removed this concept thankfully.